### ISSUE TITLE
[FEATURE] GEN 8/9

### ISSUE THREAD

**Issue by venimo** [TheArtificial/pokemon-icons#40]
Are you open for creating the last 2 gens too? Love the styling for my projects.

**Comment by yacheberries** [comment]
Hello! I'm a designer/illustrator who also really enjoys the style of these icons, and I would love to help work on the SVG images for Pokémon from generations 8/9! I sent a proposal on Upwork, along with a mock-up Grookey and shiny Grookey as a sort-of proof of concept in emulating the style!
![Grookey_CLinFG](https://github.com/TheArtificial/pokemon-icons/assets/137423575/4ac38cdc-6bdb-42f4-a22d-9e08c9b3e202)

**Comment by shankarsivarajan** [comment]
While gens 8 and 9 would be great, don't forget the mega-evolutions!

**Comment by yacheberries** [comment]
Yes, certainly! It'd be a whole lot of fun to create icons for the Mega Evolutions/Gigantamax forms as well!!

**Comment by gerwitz** [comment]
The Artificial was a design studio that stopped operations in 2018, so you can imagine won't be making any updates. But as the legacy maintainer of this repository, I'm happy to accept PRs and might even get around to updating the preview page builder when it succumbs to bitrot.

**Comment by venimo** [comment]
> Yes, certainly! It'd be a whole lot of fun to create icons for the Mega Evolutions/Gigantamax forms as well!!

Can't wait if you are able to make those icons work :D

**Comment by HighscorebreakerJoe** [comment]
So happy to see there is new hope for new icons 😄 You nailed the art style with your Grookey @yacheberries 👍
